code/_onclick/hud/screen_objects.dm

```diff
@@ -6,7 +6,7 @@
 	They are used with the client/screen list and the screen_loc var.
 	For more information, see the byond documentation on the screen_loc and screen vars.
 */
-/obj/screen
+/atom/movable/screen
 	name = ""
 	icon = 'icons/mob/screen/generic.dmi'
 	plane = HUD_PLANE
```

Comment:
Unacidable isn't a valid var for this typepath

(See below because for some reason github doesn't let me select the specific line)